salt/reviewer-suggestions/init.sls

```diff
@@ -141,9 +141,17 @@ reviewer-suggestions-cron:
         - minute: 0
         - user: {{ pillar.elife.deploy_user.username }}
 
-reviewer-suggestions-server-service-started:
+reviewer-suggestions-server-service-enabled:
     service.running:
-        - order: last
         - name: reviewer-suggestions-server
         - require:
             - file: reviewer-suggestions-server-service
+
+reviewer-suggestions-server-service-started:
+    cmd.run:
+        - order: last
```

Comment:
don't think you need `order: last` given the `require` already puts this at the end?